torchvision/csrc/macros.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(torchvision_BUILD_STATIC_LIBS)
```

Comment:
For consistency with the rest of our env variables:

```suggestion
#if defined(_WIN32) && !defined(TORCHVISION_BUILD_STATIC_LIBS)
```